README.md

```diff
@@ -1,141 +1,146 @@
+Initially, all votes will last 1 Month, require 50% approval and 1% approval quorum. Creating votes requires depositing 1000 ANT into a *proposal agreement* with the following terms:
 
-Attempts to resolve this issue by incentivizing participation in governance can have unintended consequences, for example paying people to vote will increase voter turnout but does not necessarily result in voters fairly considering all options and voting responsibility. [Futarchy](http://merkle.com/papers/DAOdemocracyDraft.pdf), in theory, does an excellent job of aligning participation incentives so long as there is an objectively measurable outcome to optimize.
+> Proposals must be made in good faith with the intention to improve the network's operational efficiency, quality, or breadth of service, and benefit all ANT holders in equal measure.
 
-However, in the absence of a measurable outcome we can use collateralized agreements and the court mechanism as a general primitive for incentivizing participation in governance that generally aligns with the values of a community. In order to participate in certain aspects of governance we can require agents to collateralize an agreement with ANT, committing them to a specific code of conduct. In the event that a decision is made that violates the code of conduct, such as voting for a proposal to send all treasury funds to participants who vote to approve the proposal, anyone can pay an arbitration fee to raise a dispute and put the offending parties collateral at risk. In exchange for taking on the risk and putting up collateral, the network can use monetary policy to incentivize participation without encouraging participants to vote in a malicious or negligent way.
+The organization will face many operational decisions, including but not limited to the following:
 
-# 6. Conclusion
-We propose the Aragon Network, an opt-in blockchain native jurisdiction, as a means to improve social scalability in the blockchain ecosystem without requiring users to give up their privacy and freedom by associating their physical identity with their blockchain identity. Economic incentives are used to create a self-reinforcing subjective oracle for enforcing human readable agreements and resolving disputes. Since the network does not rely on physical identities to enforce agreements users are required to collateralize their agreements with digital assets, which can be highly inefficient compared to traditional jurisdictions. To minimize capital costs of collateralized agreements, the networks native digital asset ANT is designed to be an efficient and practical reserve currency that retains its primary utility, governance, even while used as collateral.
+1. Changes to the organization's permissions or the content of the proposal agreement itself.
+2. Changes to court fees and other configuration parameters.
+3. Managing a treasury which is funded via a portion of the Court fees.
```

Comment:
Will forking be addressed (if it needs to be)?

Reply:
I think forking and also the futarchy fork choice rule represent an improvement to the final appeal mechanics, which are not currently within the scope of the initial court mvp. I think they should definitely be included in a subsequent revision.